library/lms.c

```diff
@@ -516,17 +523,18 @@ static int get_merkle_path( mbedtls_lms_private_t *ctx,
     {
         adjacent_node_id = curr_node_id ^ 1;
 
-        memcpy( &path[height * MBEDTLS_LMS_M_NODE_BYTES(ctx->params.type)],
-                &tree[adjacent_node_id],
-                MBEDTLS_LMS_M_NODE_BYTES(ctx->params.type) );
+        memcpy( &path[height * node_bytes],
+                &tree[adjacent_node_id * node_bytes], node_bytes );
```

Comment:
Why the second parameter (source address) has been changed?
`&tree[adjacent_node_id]` -> `&tree[adjacent_node_id * node_bytes]`

Reply:
The type of tree changed, now it's a flat unsigned char array.